test/screenshotter/ss_data.yaml

```diff
@@ -148,6 +148,8 @@ UnsupportedCmds:
     noThrow: 1
     errorColor: "#dd4c4c"
     nolatex: deliberately does not compile
+Verb:
+    \verb \verb , \verb|\verb|, \verb!<x> & </y>!
```

Comment:
Now that we have ␣, there should be a `\verb*` in there as well. Perhaps use an array environment to provide multiple lines of content in a single screenshot.